documentation/src/main/asciidoc/userguide/chapters/statistics/Statistics.adoc

```diff
@@ -0,0 +1,159 @@
+`getCollectionRoleNames`:: Get the names of all collection roles configured with the current `SessionFactory`.
+
+[[statistics-session]]
+==== Session statistics methods
+
+`getSessionCloseCount`:: Global number of sessions that got closed.
+`getSessionOpenCount`:: Global number of sessions that got opened.
+`getFlushCount`:: Get the global number of flush executed by sessions (either manual or automatic).
+
+[[statistics-jdbc]]
+==== JDBC statistics methods
+
+`getPrepareStatementCount`:: The number of JDBC prepared statements that were acquired by Hibernate.
+`getCloseStatementCount`:: The number of JDBC  prepared statements that were released by Hibernate.
+`getConnectCount`:: Get the global number of connections acquired by the Hibernate Sessions (the actual number of connections used may be much smaller depending whether you use a connection pool or not).
+
+[[statistics-transaction]]
+==== Transaction statistics methods
+
+`getSuccessfulTransactionCount`:: The number of transactions that completed successfully.
+`getTransactionCount`:: The number of transactions we know to have completed.
+
+[[statistics-concurrency-control]]
+==== Concurrency Control statistics methods
+
+`getOptimisticFailureCount`:: The number of Hibernate `StaleObjectStateException`s or JPA `OptimisticLockException`s that occurred.
+
+[[statistics-entity]]
+==== Entity statistics methods
+
+`getEntityDeleteCount`:: Get the global number of entity deletes.
+`getEntityInsertCount`:: Get the global number of entity inserts.
+`getEntityLoadCount`:: Get the global number of entity loads.
+`getEntityFetchCount`:: Get the global number of entity fetches.
+`getEntityUpdateCount`:: Get the global number of entity updates.
+
+[[statistics-collection]]
+==== Collection statistics methods
+
+`getCollectionLoadCount`:: Global number of collections that were loaded.
+`getCollectionFetchCount`:: Global number of collections that were fetched.
+`getCollectionUpdateCount`:: Global number of collections that were updated.
+`getCollectionRemoveCount`:: Global number of collections that were removed.
+`getCollectionRecreateCount`:: Global number of collections that were recreated.
+
+[[statistics-query]]
+==== Query statistics methods
+
+`getQueryExecutionCount`:: Get the global number of executed queries.
+`getQueryExecutionMaxTime`:: Get the time in milliseconds of the slowest query.
+`getQueryExecutionMaxTimeQueryString`:: Get the query string for the slowest query.
+
+[[statistics-natural-id]]
+==== Natural Id statistics methods
+
+`getNaturalIdQueryExecutionCount`:: Get the global number of naturalId queries executed against the database.
+`getNaturalIdQueryExecutionMaxTime`:: Get the global maximum query time for naturalId queries executed against the database.
+`getNaturalIdQueryExecutionMaxTimeRegion`:: Get the region for the maximum naturalId query time.
+`getNaturalIdQueryExecutionMaxTimeEntity`:: Get the entity for the maximum naturalId query time.
```

Comment:
You wrote it `Natural Id` in the title, let's try to be consistent.

Reply:
Fixed.